ios/browser/brave_wallet/brave_wallet_factory_wrappers.mm

```diff
@@ -37,75 +37,86 @@ + (nullable id)serviceForBrowserState:(ChromeBrowserState*)browserState {
 
 @implementation BraveWalletBitcoinWalletServiceFactory
 + (nullable id)serviceForBrowserState:(ChromeBrowserState*)browserState {
-  auto service = brave_wallet::BitcoinWalletServiceFactory::GetForBrowserState(
-      browserState);
-  if (!service) {
+  auto* brave_wallet_service =
+      brave_wallet::BraveWalletServiceFactory::GetServiceForState(browserState);
+  if (!brave_wallet_service) {
     return nil;
   }
+  mojo::PendingRemote<brave_wallet::mojom::BitcoinWalletService> pending_remote;
+  brave_wallet_service->Bind(pending_remote.InitWithNewPipeAndPassReceiver());
   return [[BraveWalletBitcoinWalletServiceMojoImpl alloc]
```

Comment:
I've reverted changes of bitcoinWalletService being optional for ios. It seems we don't need this and it's ok to create BraveWalletBitcoinWalletServiceMojoImpl from disconnected pending_remote - it just would fail for any call and we are not going to call it.